events/minecraft/message.js

```diff
@@ -146,6 +146,32 @@ module.exports = {
                     files: [new MessageAttachment(generateMessageImage(formattedMessage), 'guildEvent.png')],
                 });
             }
+            if (config.webhook.enabled) {
+                const event = determineEvent(msgString);
+                if (Object.keys(config.webhook.events).includes(event)) {
+                    if (config.webhook.events[event]) {
+                        let payload = {
+                            'player': msgString.split(' ')[0].includes("MVP") || msgString.split(' ')[0].includes("VIP") ? msgString.split(' ')[1] : msgString.split(' ')[0],
+                        }
+                        switch (event) {
+                            case 'guildPromote':
+                                payload['rank'] = msgString.split(' ')[msgString.split(' ').length - 1];
+                                break;
+                            case 'guildDemote':
+                                payload['rank'] = msgString.split(' ')[msgString.split(' ').length - 1];
+                                break;
+                            case 'guildMute':
+                                payload['player'] = msgString.split(' ')[msgString.split(' ').length - 3]
+                                payload['duration'] = msgString.split(' ')[msgString.split(' ').length - 1];
+                                break;
+                            case 'guildUnmute':
+                                payload['player'] = msgString.split(' ')[msgString.split(' ').length - 1];
+                                break;
+                        }
+                        await axios.post(config.webhook.url, {event, payload}, {headers: {'Authorization': config.webhook.auth}});
```

Comment:
In the readme file it says it is recommended to set api key but does not state that it is required

Reply:
This was just as an option in order to keep their api private. its not required but is recommended